vignettes/sliding.Rmd

```diff
@@ -0,0 +1,281 @@
+---
+title: "Demonstrations of sliding AR and ARX forecasters"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Demonstrations of sliding AR and ARX forecasters}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>",
+  warning = FALSE,
+  message = FALSE
+)
+```
+
+```{r pkgs}
+library(epipredict)
+library(epiprocess)
+#library(covidcast)
+library(data.table)
+library(dplyr)
+library(tidyr)
+library(ggplot2)
+```
+
+# Demonstrations of sliding AR and ARX forecasters
+
+An key function from the epiprocess package is `epi_slide()`, which allows the user to apply a function or formula-based computation an `epi_df` over a running window of `n` time steps that may or may not be grouped (see [`epiprocess` "Slide a computation over signal values" vignette](https://cmu-delphi.github.io/epiprocess/articles/advanced.html#version-aware-forecasting-revisited-1) to go over the basics of this function). The equivalent sliding method for an `epi_archive` object can be called by using the wrapper function `epix_slide()` The key difference from `epi_slide()` is that it performs version-aware computations. That is, the function only uses data that would have been available as of time t, for that reference time. 
```

Comment:
"A key function..."

* Maybe also mention why this is useful here? (backtesting our forecaster)

Reply:
There's still a few typos in this sentence.

Reply:
Should be fixed now.